mne/io/meas_info.py

```diff
@@ -1007,6 +1007,7 @@ def _update_redundant(self):
             self['ch_names'] = [ch['ch_name'] for ch in self['chs']]
             self['nchan'] = len(self['chs'])
 
+    @deprecated('use inst.pick_channels instead.')
```

Comment:
This should just automagically work. If it doesn't, I'll push a commit to fix the decorator (it's possible it has some bug)

Reply:
CIs are happy, so all good I guess, but I don't get any warning issued when using a function decorated with `@deprecated` in python console or in Spyder's IPython console. The docstring is edited however with `.. warning:: DEPRECATED:`. Is this the correct behavior for this decorator?

Reply:
I cannot replicate; on your branch I get:
```
>>> import mne
>>> import warnings
>>> warnings.simplefilter('always', DeprecationWarning)
>>> info = mne.create_info(2, 1000., 'eeg')
>>> info.pick_channels(info['ch_names'][:1])
<decorator-gen-25>:4: DeprecationWarning: Function pick_channels is deprecated; use inst.pick_channels instead.
```

Reply:
Getting the warning as well on this mac laptop.. I guess the problem is with my windows computer and the venv.